FLEDGE.md

```diff
@@ -83,7 +83,7 @@ Browsers keep track of the set of interest groups that they have joined.  For ea
 
 ```
 const myGroup = {
-  'owner': 'www.example-dsp.com',
+  'owner': 'https://www.example-dsp.com',
```

Comment:
Why include this at all? Isn't it redundant with the origin of `biddingLogicUrl`

Reply:
I defer to @michaelkleber if there's any case where we want to support this.  With decision logic it's particularly problematic, since publish, seller, and decision logic could be 3 different origins, and there's no such thing as 3-party CORS.

Could imagine signed exchanges, I guess, where the seller is the party that signed it, and the URL is where to fetch the scripts (same for bidders, of course)

Reply:
I agree that including `owner` as its own field is redundant because it needs to match the origin of some other URLs, but "IG owner" is an important enough attribute that I think it's sensible to specify it explicitly anyway.